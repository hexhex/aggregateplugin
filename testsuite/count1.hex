count(X) :- &count[p,mask,x,mask](X).
p(a,x,u).
p(b,x,v).
p(c,y,w).
q(e).
q(f).
g(u).
