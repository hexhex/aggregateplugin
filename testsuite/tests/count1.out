{g(u), q(f), q(e), p(c,y,w), p(b,x,v), p(a,x,u), count(2)}
